src/main/java/ch/njol/skript/expressions/ExprGravity.java

```diff
@@ -60,14 +60,18 @@ protected String getPropertyName() {
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if (mode == ChangeMode.SET || mode == ChangeMode.RESET)
+		if (mode == ChangeMode.SET || mode == ChangeMode.RESET || mode == ChangeMode.TOGGLE)
 			return new Class[] {Boolean.class};
 		return null;
 	}
 	
 	@Override
 	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) throws UnsupportedOperationException {
-		for (final Entity entity : getExpr().getArray(e))
-			entity.setGravity(delta == null ? true : (Boolean) delta[0]);
+		if (mode == ChangeMode.TOGGLE)

```

Comment:
I think you should put curly brackets on the if / else statements here. *Maybe* the for loops too.